front/src/store/index.js

```diff
@@ -20,6 +20,9 @@ export default new Vuex.Store({
     }
   },
   actions: {
+    signUp ({ commit }, body) {
+      return axios.post('https://localhost:5001/api/auth/signup', body);
```

Comment:
put "https://localhost:5001" to config, don't hardcode it

Reply:
okay

Reply:
@demosglok Maybe I should make the webpack settings?